redhat-developer/app-services-cli
# feat: added name flag to connector delete

Closes # 

Added `--name` flag in connector delete so either Id, name or context can be used to delete connector

### Verification Steps

1. Have a connector called `X`
2. Run `rhoas connector delete --name=X`
3. Check if connector is deleted

### Type of change
